apps/reference/docs/guides/auth.mdx

```diff
@@ -106,9 +108,9 @@ We provide a suite of Providers and login methods.
   </div>
 </div>
 
-### Simple interface
+### Easily configure third-party providers
```

Comment:
style nit: We should avoid using words like "simple" and "easy" as that is interpreted in different ways depending on the user.

```suggestion
### Configure third-party providers
```